chain-vote/src/tally.rs

```diff
@@ -50,7 +50,7 @@ impl From<(&ElectionPublicKey, &Crs)> for ElectionFingerprint {
 pub struct EncryptedTally {
     r: Vec<Ciphertext>,
     fingerprint: ElectionFingerprint,
-    max_stake: u64,
+    pub max_stake: u64,
```

Comment:
I thought that was what you were suggesting on the `pub fn max_stake(&self) -> u64` above. I guess this way makes me a bit uneasy, since it makes it possible to corrupt the internal state, was there something else you had in mind? FWIW, the only use of the field outside the module is a single function in `ProposalManager`